tests/test_input.py

```diff
@@ -394,15 +397,19 @@ def test_measure(input_args):
     simulation_config_dict = config_dict["simulation"][args.simulation]
     simulation_config_dict["max_number"] = 6
     simulation_config_dict["batch_size"] = 4
+    batch_size = simulation_config_dict["batch_size"]
+    stamp_size = simulation_config_dict["stamp_size"]
+    survey_name = simulation_config_dict["survey_name"]
+
     user_config_dict = config_dict["user_input"]
     catalog_name = os.path.join(
         user_config_dict["data_dir"], simulation_config_dict["catalog"]
     )
     # Set parameter values in param
-    param = btk_input.get_config_class(
-        simulation_config_dict, catalog_name, args.verbose
-    )
-    basic_meas(param, user_config_dict, simulation_config_dict, btk_input)
+    # param = btk_input.get_config_class(
+    #     simulation_config_dict, catalog_name, args.verbose
+    # )
```

Comment:
I think you can remove these lines completely since `param` since these returns `btk.config.Simulation_params` which we are not using anymore.

Reply:
Yes, I only commented them temporarily to remember they were there ; I will remove them once I finish working on those tests.